src/main/resources/com/sap/sgs/phosphor/fosstars/tool/github/MarkdownReporterMainTemplate.md

```diff
@@ -1,5 +1,16 @@
+# Statistics
+
+|                 | # or projects             |  % or projects              |
+| :-------------- | ------------------------: | --------------------------: |
+| Total           |      %NUMBER_OF_PROJECTS% |                        100% |
+| BAD rating      |      %NUMBER_BAD_RATINGS% |      %PERCENT_BAD_RATINGS%% |
+| MODERATE rating | %NUMBER_MODERATE_RATINGS% | %PERCENT_MODERATE_RATINGS%% |
+| GOOD rating     |     %NUMBER_GOOD_RATINGS% |     %PERCENT_GOOD_RATINGS%% |
+| Unclear ratings |  %NUMBER_UNCLEAR_RATINGS% |  %PERCENT_UNCLEAR_RATINGS%% |
+| Unknown ratings |  %NUMBER_UNKNOWN_RATINGS% |  %PERCENT_UNKNOWN_RATINGS%% |
+
 # Projects
 
-| Project | Stars | Score | Rating | Confidence | Last updated |
-| ------- | ----- | ----- | ------ | ---------- | ------------ |
+| Project | Stars | Score<br>from&nbsp;0&nbsp;to&nbsp;10 | Rating | Confidence | Last<br>updated |
```

Comment:
`Score from 0 to 10` column seems to be right aligned in the report, https://github.com/artem-smotrakov/fosstars-rating-core/blob/better-report/docs/oss/security/README.md

Reply:
Exactly, that's how it is supposed to be.